miniapps/meshing/mesh-explorer.cpp

```diff
@@ -1250,6 +1252,25 @@ int main (int argc, char *argv[])
          cout << "New mesh file: " << omesh_file << endl;
       }
 
+      if (mk == 'D')
+      {
+         const char mesh_prefix[] = "mesh-explorer.mesh.";
```

Comment:
We could ask for a filename, with `mesh-explorer.mesh` being the default if the user enters an empty string. Otherwise, the user has to rename many files (one per rank) and would need to make their own little script to do that.

Reply:
Good point. I added a prompt for mesh prefix as you suggested.